### ISSUE TITLE
Discuss releasing Recon-Act artifacts on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [inclusionAI/AWorld#502]
Hi @chengle 🤗

Niels here from the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2509.21072.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim
the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw in your GitHub README (https://github.com/inclusionAI/AWorld/tree/main/examples/visualwebarena) that the code for Recon-Act will be released soon. That's fantastic news! We would be thrilled to help you host your Recon-Act system's model checkpoints and any associated datasets on the 🤗 Hub once they are ready.

It'd be great to make these artifacts available on the 🤗 hub, to improve their discoverability/visibility.
We can add tags so that people find them when filtering https://huggingface.co/models and https://huggingface.co/datasets. For your browser-use system, a `pipeline_tag` like `image-text-to-text` (for vision-language models for computer use agents) or `robotics` could be very suitable.

## Uploading models

See here for a guide: https://huggingface.co/docs/hub/models-uploading.

In this case, we could leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, one can leverages the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

We encourage researchers to push each model checkpoint to a separate model repository, so that things like download stats also work. We can then also link the checkpoints to the paper page.

## Uploading dataset

If you also plan to release any specific data, perhaps related to the VisualWebArena examples or generated tools, it would be awesome to make that available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

Let me know if you're interested/need any help regarding this once your code is released!

Cheers,

Niels
ML Engineer @ HF 🤗

**Comment by kevinhikali** [comment]
Hi NielsRogge,
I'm a member of the AWorld team. Thank you very much for your interest in our project.
We're happy to share our work. ReconAct is currently undergoing an intensive code refactoring, which is expected to take some time. Once completed, we'll eagerly release the inference code. For the model and dataset, we use the GPT-5-chat model, and the trajectory data is based on the test set of visualwebarena. We are currently planning some work on the production of training dataset based on ReconAct. We will release the dataset to huggingface when it's completed.

Thank you again for your interest in AWorld project. Please kindly stay tuned for more of our upcoming work.

Best wishes
